models/DataObject/ClassDefinition/Data/Multiselect.php

```diff
@@ -707,7 +707,13 @@ public function preSave($containerDefinition, $params = [])
             $context = [];
             $context['fieldname'] = $this->getName();
 
-            $options = $optionsProvider->getOptions($context, $this);
+            try{
+                $options = $optionsProvider->getOptions($context, $this);
```

Comment:
or this should be handled in the Option provider implementation? since there's no reference of exception on interface https://github.com/pimcore/pimcore/blob/10.x/models/DataObject/ClassDefinition/DynamicOptionsProvider/MultiSelectOptionsProviderInterface.php#L22-L28

Reply:
:thinking: 
not sure. The point is to prevent the method _getOptions_ to bubble errors to the calling code. Since it might fail with the scenario given in #9991. Therefore an implementation of error handling in many places would always be discouraged by a handling in a central place IMHO.
Also the error is not thrown to higher callings, it's just about not validating multiselect data if there is no data.

But I am always open to improvements. How would you try to tackle this situation?